### DISCUSSION TITLE
How do I make printing a Python string work in the Playground?

### DISCUSSION THREAD

**Discussion by TZDak** [modular/modular#104] [Q&A]
I just got access to the Playground and since I had just suggested a possible but untested workaround to someone for subscripting strings, I thought I would try something like it. So I modified the first example to read...
```
import Python
print(Python.str("Hello World!")[0:10])
print("Hello Mojo!!")
```
...and it prints "Hello Mojo!!" but seems to act like the other two lines aren't even there.
I have seen an alternate import format and figured it was required but I couldn't recall it exactly so I was hoping I would get a helpful error message that I could use to help me with that. Instead, it just runs... but acts like there's only the last line of code.

Any ideas?

**Comment by ksandvik** [comment]
Is this still happening with the updated playground runtime that was announced minutes ago? I.e. from release notes:

- The String type now supports indexing with either integers or slices.

Actually I tested it here:

```
from Python import Python
print(Python.str("Hello World!")[0:10])
print("Hello Mojo!!")
```
Hello Mojo!!

**Reply by sa-** [reply]
On a side note, where are runtime updates announced?

**Reply by ksandvik** [reply]
Just now part of the docs online, or: https://docs.modular.com/mojo/changelog.html

**Reply by strangemonad** [reply]
I believe all python types now "just work" with mojo's `print`

From the 2023-05-16 changelog "PythonObject is now [printable from Mojo](https://docs.modular.com/mojo/notebooks/HelloMojo.html#python-integration), instead of requiring you to import Python’s print function."
